src/server/req_select.c

```diff
@@ -673,6 +673,10 @@ build_selentry(svrattrl *plist, attribute_def *pdef, int perm, struct select_lis
 
 	resc_access_perm = old_perms;
 	if (rc) {
+		if (rc == PBSE_UNKRESC) {
```

Comment:
Can the decode function not return any other error like PBSE_BADATVAL etc. ?
Maybe we should add a check for rc != PBSE_NONE. What do you think?

Reply:
Got it, Thank for the explanation.